### DISCUSSION TITLE
Urgent -- Don't know how to "shut down"/sleep a VM

### DISCUSSION THREAD

**Discussion by kcentric** [ubicloud/ubicloud#1429] [Q&A] [answered]
I've used AWS in the past and have gotten into trouble by leaving a VM "on" (running) without even actively using it - got charged $54 once unknowingly.

I was told that Ubicloud only charges for "usage" - I want to confirm if I can leave my VM running without actively "using" it and not get charged? If not, how do keep a VM off / sleeping / shut down without having to delete it? I hope I don't get charged accidentally.

Thanks in advance!

**Comment by byucesoy** [comment, marked as answer]
Hi @kcentric,

Sorry for the confusion. Keeping the VM running even if you are not using actively, falls under the definition of "usage". Because even if you are not actively using it, we allocate the  resources like CPU, memory, storage for you already. Could you please point me to the place where did you read/heard this so we can clarify it there as well to prevent future confusion?

Unfortunately, currently there is no way to turn off the VM other than deleting it completely. This feature is asked few times before and we will implement it at some point, however there is no timeline yet.

**Reply by kcentric** [reply]
Oh, no worries @byucesoy. You were actually the one who told me about Ubicloud's "pay only if you use" policy - I'm the same "Krishna Tripathi" emailing you from yoyoraftaar13@gmail.com about using Ubicloud.

I experimented with VMs a bit, and I've deleted mine for now because I wasn't using it.

**Reply by byucesoy** [reply]
I see, just to understand the source of confusion, are you referring to this part in my e-mail?

> ... also our pricing model is based on minutes, that means if you only use VM for 1000 minutes we only charge you (or use the credits) for that 1000, not the whole month.

By use, I meant keeping the VM up here, but I can see how it can cause some misunderstanding. Thanks for the feedback, I'll be more careful with the wording.

**Reply by kcentric** [reply]
Yes, that one. Thank you 🙂
